detox/e2e/test/autocomplete/edit_channel.e2e.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {toChannelScreen} from '@support/ui/screen';
+
+import {Setup} from '@support/server_api';
+
+describe('Autocomplete', () => {
+    beforeAll(async () => {
+        const {user} = await Setup.apiInit();
+        await toChannelScreen(user);
+    });
+
+    it('should render autocomplete in channel header edit screen', async () => {
+        // # Open channel info modal
+        await element(by.id('channel.title.button')).tap();
+
+        // # Open edit channel menu
+        await element(by.text('Edit Channel')).tap();
+
+        // # Activate at_mention autocomplete
+        await element(by.id('edit_channel.header.input')).typeText('@');
+
+        // * Expect autocomplete to render
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+    });
+});
```

Comment:
@josephbaylon Do we have TM4J test cases for these? If none, those should be created first and then match the ID/title here.

Same with other test blocks.

Reply:
@saturninoabril I added the TM4J test cases and left them on `Draft` status (which can be moved to `Active` later when this is in production). I requested @avasconcelos114 to updated the tests with the TM4J keys. I also added the JIRA ticket as well as this PR link to traceability.